### ISSUE TITLE
Circle pad and C-stick not working 

### ISSUE THREAD

**Issue by steelmonkey** [CTurt/3DSController#19]
I recently tried this application and it works except for the circle pad and c-stick. All the buttons have there configured inputs working and the touch screen works for mouse but the sticks don't work. I have even tried to bind the pad and stick to mouse with no response from either. I would appreciate any help and support thank you.

**Comment by poutros** [comment]
Are you sure the sticks don't work? Have you used a compatible app that supports the joysticks?

**Comment by steelmonkey** [comment]
Yeah im tested it out in dolphin, joytokey,vJoy, and windows controller driver thing. each showed lack of response.

**Comment by ghost** [comment]
Same problem here does it work for you now?

**Comment by steelmonkey** [comment]
Yeah I reinstalled homebrew and vjoy recognized my 3ds.
